boards/x86/acrn/doc/index.rst

```diff
@@ -0,0 +1,123 @@
+.. _acrn:
+
+ACRN UOS (Guest Operating System)
```

Comment:
Nitpick: I think it's preferable to use `User` here so it matches the `UOS` acronym we use throughout this doc.
```suggestion
ACRN UOS (User Operating System)
```

Reply:
done